modules/nodes-custom-scheduler-deploying.adoc

```diff
@@ -94,8 +98,9 @@ spec:
       volumes: []
 ----
 <1> This procedure uses the `kube-system` namespace, but you can use the namespace of your choosing.
-<2> The command for your custom scheduler might require different arguments. For example, you can pass configuration as a mounted volume using the `--config` argument.
-<3> Specify the container image that you created for the custom scheduler.
+<2> Define a scheduler profile for your custom scheduler. This is used when defining the `schedulerName` in the `Pod` configuration.
+<3> The command for your custom scheduler might require different arguments. For example, you can pass configuration as a mounted volume using the `--config` argument.
```

Comment:
this is the line we'll need to change, now users *must* pass a `KubeSchedulerConfiguration` mounted as a volume in order to specify the `schedulerName`. I would call out to https://kubernetes.io/docs/reference/scheduling/config/ for reference here

So, we should update the command arguments in the scheduler pod to show `--config=custom-config.yaml` similar to what we have in the default scheduler's pod yaml:
* https://github.com/openshift/cluster-kube-scheduler-operator/blob/master/bindata/assets/kube-scheduler/pod.yaml#L39
* https://github.com/openshift/cluster-kube-scheduler-operator/blob/master/bindata/assets/kube-scheduler/pod.yaml#L50-L52
* https://github.com/openshift/cluster-kube-scheduler-operator/blob/master/bindata/assets/kube-scheduler/pod.yaml#L124-L127

Though it might be easier for users to simply create a configmap and mount that, so the above examples would use configmap-as-volume steps like https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#add-configmap-data-to-a-volume